README.md

```diff
@@ -181,40 +168,38 @@ shasum -a 256 <file>
 # If on Windows
 certutil -hashfile <file> SHA256
 ```
+#### Setting up client dependencies
+* Darkflame Universe can run with either a packed or an unpacked client.
+* Navigate to `sharedconfig.ini` and fill in the `client_path` field with the location of your client.
 
-#### Unpacking the client
-* Clone lcdr's utilities repository [here](https://github.com/lcdr/utils)
-* Use `pkextractor.pyw` to unpack the client files if they are not already unpacked
-
-#### Setup resource directory
-* In the `build` directory create a `res` directory if it does not already exist.
-* Copy over or create symlinks from `macros`, `BrickModels`, `chatplus_en_us.txt`, `names`, and `maps` in your client `res` directory to the server `build/res` directory
-* Unzip the navmeshes [here](./resources/navmeshes.zip) and place them in `build/res/maps/navmeshes`
+#### Client database
+* On the initial sever start, the server will do the following
 
-#### Setup locale
-* In the `build` directory create a `locale` directory if it does not already exist
-* Copy over or create symlinks from `locale.xml` in your client `locale` directory to the `build/locale` directory
+- Extract the file `cdclient.fdb` from your packed client (if you have an unpacked client this step is skipped.)
+- `cdclient.fdb` will be converted to `CDServer.sqlite`.
 
-#### Client database
-* Move the file `res/cdclient.fdb` from the unpacked client to the `build/res` folder on the server.
-* The server will automatically copy and convert the file from fdb to sqlite should `CDServer.sqlite` not already exist.
-* You can also convert the database manually using `fdb_to_sqlite.py` using lcdr's utilities.  Just make sure to rename the file to `CDServer.sqlite` instead of `cdclient.sqlite`.
-* Migrations to the database are automatically run on server start.  When migrations are needed to be ran, the server may take a bit longer to start.
+* Optionally, you can also convert the database manually using `fdb_to_sqlite.py` using lcdr's utilities.  Just make sure to rename the file to `CDServer.sqlite` instead of `cdclient.sqlite`.  
+* Migrations to the database are automatically run on server start.  Migrations address issues with data in the database that would normally cause bugs.  When migrations are needed to be ran, the server may take a bit longer to start.
```

Comment:
The bullet styles are mismatched, but more importantly there should be some indentation of some of these